WooCommerce/Classes/Authentication/Navigation Exceptions/NotWPAccountViewModel.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+import SafariServices
+import WordPressAuthenticator
+import WordPressUI
+
+
+/// Configuration and actions for an ULErrorViewController, modeling
+/// an error when user attempts to log in with an invalid WordPressAccount
+struct NotWPAccountViewModel: ULErrorViewModel {
+    // MARK: - Data and configuration
+    let image: UIImage = .loginNoWordPressError
+
+    let text: NSAttributedString = NSMutableAttributedString(string: Localization.errorMessage)
```

Comment:
nit: any reasons for using the mutable version?

```suggestion
    let text: NSAttributedString = .init(string: Localization.errorMessage)
```

Reply:
Nope, good point, thanks!